ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -216,9 +246,10 @@ private HelixBootstrapUpgradeUtil(String hardwareLayoutPath, String partitionLay
           null))).getClusterMap();
     }
     String clusterNameInStaticClusterMap = staticClusterMap.partitionLayout.getClusterName();
-    this.clusterName = clusterNamePrefix + clusterNameInStaticClusterMap;
-    info("Associating static Ambry cluster \"" + clusterNameInStaticClusterMap + "\" with cluster\"" + clusterName
-        + "\" in Helix");
+    clusterName = clusterNamePrefix + clusterNameInStaticClusterMap;
+    System.out.println(
```

Comment:
why change this from `info` to `sout`?

Reply:
oh, my mistake. I should change it back to `info` after debugging.